core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -144,10 +144,28 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     blockManager
   }
 
+  // Save modified system properties so that we can restore them after tests.
+  val originalArch = System.getProperty("os.arch")
+  val originalCompressedOops = System.getProperty(TEST_USE_COMPRESSED_OOPS_KEY)
+
+  def reinitializeSizeEstimator(arch: String, useCompressedOops: String): Unit = {
+    val set = (k: String, v: String) => {
```

Comment:
Minor but, just a `def`?

Reply:
Will do (not really a Scala developer!).